<ui version="4.0" >
 <class>qscintillaSearch</class>
 <widget class="QDockWidget" name="qscintillaSearch" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>79</height>
   </rect>
  </property>
  <property name="allowedAreas" >
   <set>Qt::BottomDockWidgetArea|Qt::NoDockWidgetArea|Qt::TopDockWidgetArea</set>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>3</number>
    </property>
    <property name="spacing" >
     <number>3</number>
    </property>
    <item row="1" column="4" >
     <widget class="QCheckBox" name="cbWrap" >
      <property name="toolTip" >
       <string>If it's checked then the search wraps around the end of the text.</string>
      </property>
      <property name="text" >
       <string>Wrap</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" >
     <widget class="QCheckBox" name="cbCaseSensitive" >
      <property name="toolTip" >
       <string>If it's checked then the search is case sensitive.</string>
      </property>
      <property name="text" >
       <string>Case Sensitive</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" >
     <widget class="QCheckBox" name="cbRegExp" >
      <property name="toolTip" >
       <string>If it's checked then expr is interpreted as a regular expression rather than a simple string.</string>
      </property>
      <property name="text" >
       <string>RegExp</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" >
     <widget class="QToolButton" name="tbNext" >
      <property name="toolTip" >
       <string>Search next.</string>
      </property>
      <property name="text" >
       <string>Next</string>
      </property>
      <property name="icon" >
       <iconset resource="../../resources/resources.qrc" >:/Icons/Icons/helpnext.png</iconset>
      </property>
      <property name="toolButtonStyle" >
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
      <property name="autoRaise" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="5" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Ignored</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="6" >
     <widget class="QToolButton" name="tbClose" >
      <property name="toolTip" >
       <string>Close</string>
      </property>
      <property name="icon" >
       <iconset resource="../../resources/resources.qrc" >:/Icons/Icons/buttoncancel.png</iconset>
      </property>
      <property name="autoRaise" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QLineEdit" name="leReplace" >
      <property name="toolTip" >
       <string>Replace the current selection with this string</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" >
     <widget class="QLineEdit" name="leSearch" >
      <property name="toolTip" >
       <string>Enter the string or regular expression to find.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QToolButton" name="tbPrevious" >
      <property name="toolTip" >
       <string>Search previous.</string>
      </property>
      <property name="text" >
       <string>Previous</string>
      </property>
      <property name="icon" >
       <iconset resource="../../resources/resources.qrc" >:/Icons/Icons/helpprev.png</iconset>
      </property>
      <property name="toolButtonStyle" >
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
      <property name="autoRaise" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QToolButton" name="tbReplace" >
      <property name="toolTip" >
       <string>Replace next.</string>
      </property>
      <property name="text" >
       <string>Replace</string>
      </property>
      <property name="icon" >
       <iconset resource="../../resources/resources.qrc" >:/Icons/Icons/editreplace.png</iconset>
      </property>
      <property name="toolButtonStyle" >
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
      <property name="autoRaise" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2" >
     <widget class="QToolButton" name="tbReplaceAll" >
      <property name="toolTip" >
       <string>Replace all.</string>
      </property>
      <property name="text" >
       <string>Replace All</string>
      </property>
      <property name="icon" >
       <iconset resource="../../resources/resources.qrc" >:/Icons/Icons/filefind.png</iconset>
      </property>
      <property name="toolButtonStyle" >
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
      <property name="autoRaise" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="4" >
     <widget class="QCheckBox" name="cbWholeWords" >
      <property name="toolTip" >
       <string>If it's checked then the search looks for whole word matches only, otherwise it searches for any matching text.</string>
      </property>
      <property name="text" >
       <string>Whole Words</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5" colspan="2" >
     <widget class="QCheckBox" name="cbFromStart" >
      <property name="text" >
       <string>From Start</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>tbClose</sender>
   <signal>clicked()</signal>
   <receiver>qscintillaSearch</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>699</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel" >
     <x>60</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
